### PR TITLE
Add checker for precompile

### PR DESCRIPTION
**Pull request by AsceticBear**

### CODE REVIEW

File: runtime/crab/Cargo.toml [hunk 1]
```diff
@@ -13,7 +13,8 @@ version     = "0.12.2"
 # crates.io
 array-bytes       = { version = "1.4" }
 codec             = { package = "parity-scale-codec", version = "2.3", default-features = false, features = ["derive"] }
-log               = { version = "0.4" }
+evm               = { version = "0.33.1", default-features = false, features = ["with-codec"] }
+log               = { version = "0.4.2" }
```
Comment (AurevoirXavier) [inline runtime/crab/Cargo.toml:17]: frame_support::log

File: runtime/crab/src/pallets/mod.rs [hunk 2]
```diff
@@ -91,8 +91,8 @@ pub use proxy::*;
 pub mod multisig;
 pub use multisig::*;
 
-pub mod evm;
-pub use evm::*;
+pub mod _evm;
```
Comment (AurevoirXavier) [inline runtime/crab/src/pallets/mod.rs:94]: evm_

File: runtime/crab/Cargo.toml (hunk 1 above)
Comment (AsceticBear) [reply runtime/crab/Cargo.toml]: done

File: runtime/crab/src/pallets/mod.rs (hunk 2 above)
Comment (AsceticBear) [reply runtime/crab/src/pallets/mod.rs]: done

**Review by AurevoirXavier — APPROVED** [review]

**Review by HackFisher — APPROVED** [review]

**Pull request merged**